### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/tutorial/response-model.md`

### PR DESCRIPTION
**Pull request by gustavovalle23**
Added translation for pt to docs/pt/docs/tutorial/response-model.md.

### CODE REVIEW

**Review by cassiobotaro — APPROVED** [review]
lgtm

File: docs/pt/docs/tutorial/response-model.md [hunk 1]
```diff
@@ -0,0 +1,482 @@
...
+!!! tip
+    Observe que os valores padrão podem ser qualquer coisa, não apenas `None`.
+
+    Eles podem ser uma lista (`[]`), um `float` de `10.5`, etc.
+
+### `response_model_include` and `response_model_exclude`
+
+Você também pode usar os parâmetros do *decorador de operação de caminho* `response_model_include` e `response_model_exclude`.
+
+Eles pegam um `conjunto` de `str` com o nome dos atributos a incluir (omitindo o restante) ou a excluir (incluindo o restante).
```
Comment (fabioueno) [inline docs/pt/docs/tutorial/response-model.md:434]: I'm not sure if this has been discussed somewhere, but I wouldn't translate `set` in this case, as it's a type - just as `list` wasn't translated in line 77:

```suggestion
Eles pegam um `set` de `str` com o nome dos atributos a incluir (omitindo o restante) ou a excluir (incluindo o restante).
```

I'd suggest doing it when it's not formatted as code. 😉

File: docs/pt/docs/tutorial/response-model.md [hunk 2]
```diff
@@ -0,0 +1,482 @@
...
+
+Isso pode ser usado como um atalho rápido se você tiver apenas um modelo Pydantic e quiser remover alguns dados da saída.
+
+!!! tip
+    Mas ainda é recomendável usar as ideias acima, usando várias classes, em vez desses parâmetros.
+
+    Isso ocorre porque o JSON Schema gerado no OpenAPI do seu aplicativo (e os documentos) ainda será o modelo completo, mesmo se você usar `response_model_include` ou `response_model_exclude` para omitir alguns atributos.
```
Comment (fabioueno) [inline docs/pt/docs/tutorial/response-model.md:441]: In other moments you translated "JSON Schema" to "Esquema JSON"... 👀

```suggestion
    Isso ocorre porque o Esquema JSON gerado no OpenAPI do seu aplicativo (e os documentos) ainda será o modelo completo, mesmo se você usar `response_model_include` ou `response_model_exclude` para omitir alguns atributos.
```

**Review by fabioueno — COMMENTED** [review]
Nice one!! 👏

**Pull request closed without merging**